src/server/identity/server/api_server.go

```diff
@@ -22,7 +23,7 @@ type apiServer struct {
 }
 
 // NewIdentityServer returns an implementation of identity.APIServer.
-func NewIdentityServer(env Env, public bool) identity.APIServer {
+func NewIdentityServer(env Env, public bool) (identity.APIServer, func(context.Context) error) {
```

Comment:
I guess I didn't pick up on this when you did the license server stuff... but I think it would be better to just have an EnvBootstrap method (and in main.go, collect a slice of `interface { EnvBootstrap(ctx context.Context) error }` objects that you call EnvBootstrap on).

Reply:
so I think I didn't do this last time because it would introduce a circular dependency between the main & the IdentityServer